### ISSUE TITLE
Milestone 2 feedback

### ISSUE THREAD

**Issue by andytai7** [UBC-MDS/credit_default_prediction#44]
Dear group 11, 

You guys have made improvements! Great job! Below highlight some feedback for milestone 2. In addition, things under _Suggestions:_ should be taken as suggestions and do not affect your mark. 

**Writing an analysis that uses multiple scripts: Mechanics**

Comments
Your workflow is a bit odd, such that EDA happens after your ML analysis. This is no longer exploratory.

**Writing an analysis that uses multiple scripts: Viz**

Comments
I like the spacing and the figure captions 

Expectations: Mechanics

Comments
Arushi only has two commits, should be more equal in terms of each team member's commits.

**Project organization and documentation expectations: Reasoning**

Comments
" Our model performed fairly well on test data set with the f1 score being ~0.53." is not entirely convincing. In general, give a well or bad assessment more objectively.

There are no references for README.Rmd

Please define "default payment" what is it? (I'm not a banker) I thought defaulting meant going bankrupt (so I can't make any payments)

"most features show very little overlap" I disagree; many distributions look almost identical

Fig 5 is referred to as Fig 3; it is not clear to me which are the best hyperparameters

_Suggestions:_
Files don't all have to have 'credit_default' in the name (probably better if they don't)

Overall good work in making the analysis, but mind the interpretation of your values. Also, explain your metrics in layman's terms for the public.

**Submission expectations: Mechanics**

Comments
Why are there two readme-files in the root directory? 

Overall great job!

**Comment by Arushi282** [comment]
Hi @andytai7, 

Thank you for your feedback and we will work on making the changes you mentioned.

I just wanted to clarify one point about the Readme, you mentioned that we didn't include reference in our Rmd file, but we mention our references in the .bib file and that file is also mentioned in our Rmd at top. The references render in our Readme.md file.  Would request you to reconsider or elaborate more on this point. Thank you!

**Comment by andytai7** [comment]
Hi @Arushi282,

I would suggest adding the list also in the RMD file. 

Best, 
Andy

**Comment by liannah** [comment]
Hi @andytai7 ,

Thank you very much for the response. We will definitely work more on that. 

To elaborate a little more about ReadMe, we used readme.Rmd for rendering our .md file, because we are getting the results, such as f1 score, from the model, thus we need to render in order to see the values. 

That is why we have two files in our repository, one .md for people to read, and one .Rmd to see where the results are coming from. As for the bibliography, bibliography was used to make sure the formatting of the text in BibTex format is correct.
